docs/install/uninstall.md

```diff
@@ -0,0 +1,78 @@
+# Uninstall
+
+This article walks you through how to uninstall your Coder server.
+
+<blockquote class="danger">
+  <p>
+  Uninstalling Coder with the built-in PostgreSQL database will delete the database engine and database. Consider backing up the database directory if you would like to reuse it with a future installation. This does not apply if you have installed your own PostgreSQL database instance.
+  </p>
+</blockquote>
```

Comment:
@sharkymark is this true? I thought the database will remain even after you remove the binary/package?

Reply:
@bpmct where else do you think it stores data? there's a data dir.

When can this PR get approved? @ammario 

<img width="935" alt="image" src="https://user-images.githubusercontent.com/2022166/187512405-ef8e4d5f-4aac-48be-b077-39f41373db3d.png">

Reply:
Ah, I mean if you manually delete the `postgres` folder it removes the DB, but I consider uninstalling deleting the binary or using `apt remove`. I personally find this warning a bit confusing but we can keep or clarify